mcstatus/querier.py

```diff
@@ -96,7 +96,8 @@ class Players:
         max: int
         names: List[str]
 
-        def __init__(self, online, max, names):
+        # TODO: It's a bit weird that we accept str for number parameters, just to convert them in init
```

Comment:
Probably because of how they are initially parsed during network reads. If you want to hunt down those locations and convert to `int` beforehand, I'd be fine with that. I'd consider this an internal class that we have control over.

Reply:
yeah, I certainly could fix this, but I'm not sure if it's worth it to do that here considering there's #306 which refactors all of this and in that process, this issue was actually fixed there. (Because it's using a dataclass for this, which has these type-hinted as ints, and there's an alternative constructor taking those strings and handling the conversion outside of init).